core/src/test/scala/unit/kafka/server/KafkaRaftServerTest.scala

```diff
@@ -39,6 +39,8 @@ class KafkaRaftServerTest {
     val configProperties = new Properties
     configProperties.put(KafkaConfig.ProcessRolesProp, "broker,controller")
     configProperties.put(KafkaConfig.NodeIdProp, nodeId.toString)
+    configProperties.put(KafkaConfig.AdvertisedListenersProp, "PLAINTEXT://127.0.0.1:9092")
```

Comment:
It shouldn't be necessary to set these properties here, right? The defaults should work?